docs/examples/index.rst

```diff
@@ -16,5 +16,6 @@ This is a list of example notebooks demonstrating the use of jupyter_rfb:
     pygfx2.ipynb
     pygfx3.ipynb
     video.ipynb
+    visibility_check.ipynb
```

Comment:
Would it be worth it to do do `:glob:` at the top of the toctree and then do `*.ipynb`? I think that is supposed to work.

Reply:
Nice, it does! (except only with `*`, not with `*.ipynb`)